### ISSUE TITLE
Minimalistic homeserver implementation

### ISSUE THREAD

**Issue by turt2live** [turt2live/matrix-wishlist#5]
Ported from https://github.com/turt2live/waffle-matrix/issues/27

Preferably in node.

For the true bridge/gateway support.

Bridge - `@travis:instagram.com`
Gateway - Log in as `@travis:instagram.com` in Riot

**Comment by maxidor** [comment]
Isn't node like 300MB+ just to run something?

**Comment by turt2live** [comment]
Not really. It's a relatively small footprint. Node is easier to maintain and get community involvement on, however. The implementation could be in any language though.

**Comment by turt2live** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Minimalistic homeserver implementation](https://trello.com/c/J5NWoa6a/12-minimalistic-homeserver-implementation)
